app/views/hyrax/file_sets/_actions.html.erb

```diff
@@ -1,42 +1,44 @@
-<div class="btn-group">
+<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id)  || can?(:download, file_set.id) %>
```

Comment:
Is there a presenter available here that would allow this to be encapsulated in the presenter?  Something like...
```
presenter.can_perform_any_action?(file_set.id)
```